_docs_integrate/share-attribute-with-peer.md

```diff
@@ -15,52 +17,49 @@ properties:
-| Relationship   | Third Party     |     ✓     | `USER DECISION` / `NOT ALLOWED` (depending on confidentiality) | A user can share RelationshipAttributes of any Relationship with any other Relationship (if the confidentiality of the RelationshipAttribute is protected or public).<br> Example: Share customer ID from company A with company B (Company A is owner of RelationshipAttribute), e.g. Payback number. |
-| Relationship   | `<empty>`       |     ✓     | `AUTO ACCEPT`                                                  | An empty owner defaults to an Attribute with `owner=<Sender>`.                                                                                                                                                                                                                                         |
+| Type and context                                                                                        | Owner       |                 Possible?                  | Automation      | Remarks, reasons and examples                                                                                                                                                                                                                                                                                                                                                                                                          |
+| ------------------------------------------------------------------------------------------------------- | ----------- | :----------------------------------------: | --------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| IdentityAttribute                                                                                       | Sender      |                     ✓                      | `AUTO_ACCEPT`   | Example: A company sends a new customer its address.                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
"Example: A company sends a new customer its address." For me it is not clear if "its" refers to the company or the customer in this sentence.